pkg/ccl/importccl/import_stmt.go

```diff
@@ -400,12 +400,6 @@ func importPlanHook(
 				return err
 			}
 
-			// The IMPORT INTO prototype currently breaks secondary indexes in the
```

Comment:
Deletion was necessary to get tests running. The presence of PKs would trigger this error. I considered putting it behind a testing knob, but hopefully the PR preventing shadowing will get merged soon enough?